esite/home/apps.py

```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
+class HomeConfig(AppConfig):
```

Comment:
Please describe what this part of your code does.

Reply:
yikes xD